### PR TITLE
Cub dataset

### PR DESCRIPTION
**Pull request by mridulk97**

### CODE REVIEW

File: analysis/tsne.py [hunk 1]
```diff
@@ -337,8 +342,10 @@ def main(configs_yaml):
     dataloader = DataLoader(dataset.data, batch_size=batch_size, num_workers=num_workers, collate_fn=custom_collate)
     
     # Load model
-    if isOriginalVQGAN:
+    if isOriginalVQGAN=='VQGAN':
         model_type=VQModel
+    elif isOriginalVQGAN=='CW':
```
Comment (elhamod) [inline analysis/tsne.py:347]: before you check this in, make sure all yamls that use isOriginalVQGAN have been updated properly

File: taming/modules/vqvae/quantize.py [hunk 2]
```diff
@@ -91,12 +91,13 @@ def forward(self, z):
 
     def get_codebook_entry(self, indices, shape):
         # shape specifying (batch, height, width, channel)
```
Comment (elhamod) [inline taming/modules/vqvae/quantize.py]: Could you give a little more context why we need this? we generally use VectorQuantizer2 not VectorQuantizer.

**Review by elhamod — CHANGES_REQUESTED** [review]

File: analysis/specimen_diff_vqgan_new.py [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    # plt_path = configs_yaml.plt_path
+    
+    bb_model_path = configs_yaml.bb_model_path
+        
+    plt_path = "default"
+    
+    # phylo_distances = parse_phyloDistances(phyloDistances_string)
+    
+    # load image
+    dataset = CustomDataset(size, file_list_path, add_labels=True)
```
Comment (elhamod) [inline analysis/specimen_diff_vqgan_new.py]: Please rename this file to something like /home/elhamod/projects/phylonn/analysis/specimen_translation.py and /home/elhamod/projects/phylonn/analysis/specimen_translation_vqgan.py

File: taming/modules/vqvae/quantize.py (hunk 2 above)
Comment (mridulk97) [reply taming/modules/vqvae/quantize.py]: retraced to the original code

**Review by elhamod — CHANGES_REQUESTED** [review]

File: analysis/specimen_diff_vqgan_new.py (hunk 3 above)
Comment (elhamod) [reply analysis/specimen_diff_vqgan_new.py]: same goes for the yaml file

Comment (elhamod) [reply analysis/specimen_diff_vqgan_new.py]: and if the same original file can be used without duplicating the file, would be great

Comment (mridulk97) [reply analysis/specimen_diff_vqgan_new.py]: adding new argument for that

Comment (mridulk97) [reply analysis/specimen_diff_vqgan_new.py]: done

**Review by elhamod — APPROVED** [review]

**Pull request merged**